eve/workers/compile_centos_8/Dockerfile

```diff
@@ -21,13 +25,13 @@ RUN yum install -y \
         zlib-devel
 
 # Installation of golang
-ARG GO_VERSION=1.14.2
+ARG GO_VERSION=1.14
 RUN curl -s https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz | \
        tar -C /usr/local -xzf - && \
     ln -snf /usr/local/go/bin/go /usr/bin
 
 # Linter
-ARG GOLINT_VERSION=1.23.8
+ARG GOLINT_VERSION=1.21.0
```

Comment:
why the revert ?

Reply:
same here, I took the version number from the repository scality/hdcontroller in the dockerfile for centos8 to have the exact same version of linter for both repositories.